### ISSUE TITLE
Does importing the library fix the torch.initial_seed() ?

### ISSUE THREAD

**Issue by dlfqhsdugod1106** [HazyResearch/flash-fft-conv#26]
Hello; thanks for the great library!

I realized that `flashfftconv` seems to fix the internal seed that `pytorch` uses.
When we run the following,
```python
import torch
print(torch.initial_seed())
```
it prints some large Long seed numbers, e.g., `1885703015682584485` and `10761560373843692808`.
However, when I imported `flashfftconv`, the `initial_seed()` gave the exact same value of `23` every time.

```python
import torch
import flashfftconv
print(torch.initial_seed()) # always prints 23
```

Is this an intended behavior? 

I found this while running some simple experiments multiple times; my previous loss curves had some variance, which got disappeared once I started using the `flashfftconv`. As the seed got fixed, the dataloaders returned the same training batch every time.

Thanks!

EDIT: Indeed, I found `torch.manual_seed(23)` in `flashfftconv/conv.py`. I removed this line (in my local package), and the issue no longer persists.

**Comment by KeAWang** [comment]
I ran into this same issue and it took me a day to figure out the issue was coming from just instantiating `FlashFFTConv` despite never using it in my computations. It caused a significant change in my training (probably because of data shuffling).

@DanFu09 can you patch this to prevent other people from getting this silent error in the future?

**Comment by DanFu09** [comment]
Sorry about that, it should be fixed now!

**Comment by KeAWang** [comment]
Thanks for the quick fix!
